SCENARIOS.txt

```diff
@@ -6,5 +6,6 @@ immutable
 input-validation
 library-test
 local-scope
+random
```

Comment:
I've considered `randomness` as an alternative. I picked random as it seems to best fit the other scenarios, but I could easily be swayed to go a different route.

Reply:
I think `random` is fine. I don't feel strongly about the naming and could go either way. If someone has a really strong argument for why it should be something else I'd probably be convinced.